src/symex/symex_parse_options.cpp

```diff
@@ -364,7 +364,6 @@ bool symex_parse_optionst::process_goto_program(const optionst &options)
     remove_vector(goto_model);
     // Java virtual functions -> explicit dispatch tables:
     remove_virtual_functions(goto_model);
-    // Java instanceof -> clsid comparison:
```

Comment:
I think that bit might still be missing: it seems only cbmc_parse_options.cpp has received the new exceptions processing, while symex, goto-analyzer, goto-instrument would still benefit from it.